packages/react-code-editor/src/components/CodeEditor/CodeEditor.tsx

```diff
@@ -551,6 +572,16 @@ export class CodeEditor extends React.Component<CodeEditorProps, CodeEditorState
                     <CodeEditorContext.Provider value={{ code: value }}>{customControls}</CodeEditorContext.Provider>
                   )}
                 </div>
+              }
+              {<div className={css(styles.codeEditorHeaderMain)}>{headerMainContent}</div>}
+              {!!shortcutsPopoverProps.bodyContent && (
+                <div className="pf-c-code-editor__keyboard-shortcuts">
+                  <Popover {...shortcutsPopoverProps}>
+                    <Button variant={ButtonVariant.link} icon={<HelpIcon />}>
+                      {shortcutsPopoverProps.shortcutTitleText || 'View shortcuts'}
```

Comment:
maybe you can add 'View shortcuts' as a default value for shortcutsPopoverProps.shortcutTitleText so it's explicitly called out in the docs.